src/universal/helpers/bag.ts

```diff
@@ -31,23 +35,39 @@ export function getBagResult(
       .toLowerCase()
       .includes(bagSearchAddress.toLowerCase());
 
-    return isWoonplaatsMatch && isAddressMatch;
+    return isWeesp ? isWoonplaatsMatch && isAddressMatch : isAddressMatch;
   });
 
   return result1 ?? null;
 }
 
+export function getLatLngWithAddress(
+  result: BAGSearchResult
+): LatLngWithAddress {
+  return {
+    address: result.adres,
+    ...(getLatLngCoordinates(result.centroid) ?? null),
+  };
+}
+
+export function getLatLngCoordinates(centroid: LatLngTuple) {
+  // Forced coordinates to be in the right order
```

Comment:
Geeft het eventueel omwisselen niet een andere locatie? Ik snap niet helemaal wat hier gebeurd, het grootste coordinaat word op de eerste plek geplaatst? Ziet er wel goed uit

Reply:
Precies. Kennelijk hebben ze de lat/lon volgorde in die API aangepast. Dus om het altijd goed te hebben vergelijken we ze en nemen we het grootste getal (lat) altijd als eerste.

Reply:
Ah, het wordt dus zo aangeleverd dat het grootste getal altijd de lat is. Thanks voor de toelichting